### PR TITLE
fix #295898: Deleting a StaffTypeChange causes a crash in macOS

### PR DESCRIPTION
**Pull request by mattmcclinch**
Resolves: https://musescore.org/en/node/295898.

### CODE REVIEW

File: libmscore/measure.cpp [hunk 1]
```diff
@@ -1002,7 +1002,8 @@ void Measure::remove(Element* e)
                         // st currently points to an list element that is about to be removed
                         // make a copy now to use on undo/redo
                         StaffType* st = new StaffType(*stc->staffType());
-                        staff->removeStaffType(tick());
+                        if (!tick().isZero())
+                              staff->removeStaffType(tick());
```
Comment (Jojo-Schmitz) [inline libmscore/measure.cpp]: Does that also remove a staff type change at the very beginning of a score?

Comment (mattmcclinch) [reply libmscore/measure.cpp]: The StaffTypeChange element is removed. The point is to not remove the Staff's initial StaffType.

**Comment by Jojo-Schmitz** [comment]
Outch, of course...

File: libmscore/measure.cpp (hunk 1 above)
Comment (Jojo-Schmitz) [reply libmscore/measure.cpp]: Outch, of course...

**Pull request merged**